# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jed/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- jed-0.99-18/src/misc.c.vanilla	2007-05-17 19:14:47.000000000 +0000
+++ jed-0.99-18/src/misc.c	2007-05-17 19:14:53.000000000 +0000
@@ -741,7 +741,7 @@
 
 void jed_ungetkey_wchar (SLwchar_Type wc)
 {
-   SLuchar_Type *b, buf[SLUTF8_MAX_MBLEN];
+   SLuchar_Type *b, buf[JED_MAX_MULTIBYTE_SIZE];
 
    if (NULL == (b = jed_wchar_to_multibyte (wc, buf)))
      return;
